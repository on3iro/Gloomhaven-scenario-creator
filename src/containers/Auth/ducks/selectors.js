export const getLoggedIn = state => state.auth.user.isLoggedIn;
