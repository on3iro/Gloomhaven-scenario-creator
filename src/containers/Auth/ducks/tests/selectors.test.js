import * as selectors from '../selectors';


describe('selectors', () => {
  it('should getLoggedIn information', () => {
    expect(selectors.getLoggedIn({ auth: { user: { isLoggedIn: true }}}))
      .toMatchSnapshot();
  });
});
