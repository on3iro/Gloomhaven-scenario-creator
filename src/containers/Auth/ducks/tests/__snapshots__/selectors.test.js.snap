exports[`selectors should getLoggedIn information 1`] = `true`;
